packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-your-app/aspnet/localsignout.md

```diff
@@ -0,0 +1,49 @@
+Add a `SignOut` action that accepts an `HTTP POST` and signs the user out with `CookieAuthenticationDefaults.AuthenticationType`:
+
+```csharp
+public class AccountController : Controller
+{
+    [HttpPost]
+    public ActionResult SignOut()
+    {
+        if (HttpContext.User.Identity.IsAuthenticated)
+        {
+            HttpContext.GetOwinContext().Authentication.SignOut(CookieAuthenticationDefaults.AuthenticationType);
+        }
+
+        return RedirectToAction("Index", "Home");
+    }
+}
+```
+
+Update your using statements to import the `Microsoft.Owin.Security.Cookies` namespace:
```

Comment:
Update your `using` statements to import the `Microsoft.Owin.Security.Cookies` namespace: